code/modules/shuttles/marine_ferry.dm

```diff
@@ -590,7 +590,76 @@
+			to_chat(M, SPAN_HIGHDANGER("Alarms blare throughout the bird! The fleshy hosts are definately trying something!"))
+		else
+			to_chat(M, SPAN_HIGHDANGER("Lock-on alarms and missile warnings blare from the cockpit! Oh crap!"))
+	playsound_area(shuttle_area,  'sound/effects/alert.ogg',75, echo = list(-500, -1500, 1000, 500, 0, 0.8, 5, 5), y_s_offset = 10)
+	sleep(5 SECONDS)
+	playsound(GLOB.AAgunLocation, sound_to_play, 100, sound_range = 300,  falloff = 250, echo = list(-1000, -3500, 1000, 0, 0, 1, -2000, 0.5, 6, 2), y_s_offset = 10)
+	playsound_area(shuttle_area,  sound_to_play, echo = list(-500, -2000, 1000, 500, 0, 0.8, 5, 5), y_s_offset = 10)
+	sleep(4 SECONDS) //bullet time lulz
+	for(M in shuttle_area)
+		to_chat(M, SPAN_HIGHDANGER("The ship jostles violently as explosions rock the ship!")) //xenos getting hit
+		to_chat(M, SPAN_HIGHDANGER("You feel the ship turning sharply as it adjusts its course!"))
+		shake_camera(M, 60, 2)
+	playsound_area(shuttle_area, 'sound/effects/antiair_explosions.ogg', echo = list(-500, -1500, 1000, 500, 0, 0.8, 5, 5))
+	// Shit hits the fan
+	// Everything here is and should be non lethal to even larva. Even if the RNG gods somehow smited one with all of these explosions they will still live.
+	addtimer(CALLBACK(GLOBAL_PROC, .proc/cell_explosion,pick(turfs_int),5,1,EXPLOSION_FALLOFF_SHAPE_LINEAR,null,cause), 1 SECONDS)
+	sleep(5)
+	addtimer(CALLBACK(GLOBAL_PROC, .proc/cell_explosion,pick(turfs_int),5,1,EXPLOSION_FALLOFF_SHAPE_LINEAR,null,cause), 2 SECONDS)
+	sleep(5)
+	addtimer(CALLBACK(GLOBAL_PROC, .proc/cell_explosion,pick(turfs_int),5,1,EXPLOSION_FALLOFF_SHAPE_LINEAR,null,cause), 3 SECONDS)
+	sleep(2 SECONDS)
+	addtimer(CALLBACK(GLOBAL_PROC, .proc/cell_explosion,pick(turfs_int),5,1,EXPLOSION_FALLOFF_SHAPE_LINEAR,null,cause), 2 SECONDS)
+	create_shrapnel(pick(turfs_int), rand(5,9), , ,/datum/ammo/bullet/shrapnel/light/effect/, cause)
+	s.set_up(10, 1, pick(turfs_int)) // Spark system funny
+	s2.set_up(10, 1, pick(turfs_int))
+	s3.set_up(10, 1, pick(turfs_int))
+	s.start()
+	sleep(1 SECONDS)
+	INVOKE_ASYNC(GLOBAL_PROC, .proc/flame_radius,cause, 1, pick(turfs_int), 1, 16, FLAMESHAPE_DEFAULT, null)
+	create_shrapnel(pick(turfs_int), rand(5,9), , ,/datum/ammo/bullet/shrapnel/light/effect/ver2, cause)
+	addtimer(CALLBACK(GLOBAL_PROC, .proc/cell_explosion,pick(turfs_int),5,1,EXPLOSION_FALLOFF_SHAPE_LINEAR,null,cause), 3 SECONDS)
+	new /obj/effect/spawner/gibspawner/robot(pick(turfs_int))
+	new /obj/effect/spawner/gibspawner/robot(pick(turfs_int)) // A few random ass gibs
+	s2.start()
+	playsound(GLOB.AAgunLocation, sound_to_play2, 100, sound_range = 300,  falloff = 250, echo = list(-1000, -3500, 1000, 0, 0, 1, -2000, 0.5, 6, 2), y_s_offset = 10) // making sure?
+	playsound_area(shuttle_area,  sound_to_play, echo = list(-500, -2000, 1000, 500, 0, 0.8, 5, 5), y_s_offset = 10)
+	sleep(1 SECONDS)
+	s3.start()
+	create_shrapnel(pick(turfs_int), rand(5,9), , ,/datum/ammo/bullet/shrapnel/light/effect/ver1, cause)
```

Comment:
don't use sleep in code like this, you are literally using timers so just increment the time in which they get called